client/my-sites/email/inbox/index.js

```diff
@@ -0,0 +1,18 @@
+import page from 'page';
```

Comment:
This also should be specified here: [client/my-sites/email/index.js](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/email/index.js)